### DISCUSSION TITLE
FluentCalendar as a dropdown

### DISCUSSION THREAD

**Discussion by StefanJansson** [microsoft/fluentui-blazor#1895] [General]
Nice control but it takes to much space :) 

Can it be minimized/expanded or could it be a nice feature for you to develop? 

/Stefan

**Comment by vnbaaij** [comment]
As in https://www.fluentui-blazor.net/DateTime#defaultdatepicker?

**Reply by StefanJansson** [reply]
Yes, but for selecting a month instead of a date

**Reply by StefanJansson** [reply]
One other idea was to use it with a menubutton with a menu reference but I cannot figure out how to get it to work, When clicking on the menubutton,  it's just a placeholder for the menu that is visible below the button

![image](https://github.com/microsoft/fluentui-blazor/assets/18248455/caa1151e-1691-484e-88e4-5e7a080f5106)

![image](https://github.com/microsoft/fluentui-blazor/assets/18248455/9430c5bd-471d-451a-8513-b2bfcaf13c44)

**Reply by dvoituron** [reply]
You could also use two FluentSelect with the list of years and the list of months.

**Reply by StefanJansson** [reply]
Four clicks, I'm lazy 😂

**Reply by dvoituron** [reply]
That will be almost the same:
- click on the Combobox to open the Month Selector
- click on the Year title to open the correct year (and maybe multiple clicks to navigate to the previous years)
- click on the expected month

Many existing sites already uses 2 lists. Including on mobile.

Or you can create a PR to improve this component 😊

**Reply by StefanJansson** [reply]
Not sure I match your skills but I might give it a try

**Reply by StefanJansson** [reply]
PR posted
